DatadogRUM/Sources/Instrumentation/AppHangs/FatalAppHangsHandler.swift

```diff
@@ -92,6 +92,10 @@ internal final class FatalAppHangsHandler {
 
             let realErrorDate = fatalHang.hang.startDate.addingTimeInterval(fatalHang.serverTimeOffset)
             let realDateNow = dateProvider.now.addingTimeInterval(context.serverTimeOffset)
+            var timeSinceAppStart: TimeInterval? = nil
+            if let startTime = context.launchTime?.launchDate {
+                timeSinceAppStart = realErrorDate.timeIntervalSince(startTime) * 1_000
+            }
```

Comment:
change request/ The `timeSinceAppStart` computation here is wrong:
- `fatalHang.hang.startDate` is the start date of the hang in **previous app process** (terminated one).
- `context.launchTime.launchDate` is the launch date of the **current app process** (the one sending error).

so the "time since" value will become negative.


We must persist the `appLaunchDate` information [in `FatalAppHang`](https://github.com/DataDog/dd-sdk-ios/blob/caa508b7097f0a0bdc9dab1ae668bb162c2c7fee/DatadogRUM/Sources/Instrumentation/AppHangs/AppHang.swift#L31-L34) similar way to how we do it for crash reports (in `CrashContext`).